lib/dry/auto_inject/strategies/args.rb

```diff
@@ -42,9 +44,9 @@ def initialize(#{initialize_args})
 
         def define_initialize_with_splat(super_method)
           super_params = if super_method.parameters.any? { |type, _| type == :rest }
-            '*args'
-          else
-            "*args[0..#{super_method.parameters.length - 1}]"
+                           '*args'
+                         else
+                           "*args[0..#{super_method.parameters.length - 1}]"
           end
```

Comment:
Not a great fan of RuboCop's suggestion here either. I'd probably just do it this way if I were rewriting today:

```ruby
super_params =
  if super_method.parameters.any? { |type, _| type == :rest }
    '*args'
  else
    "*args[0..#{super_method.parameters.length - 1}]"
  end
```